src/workloads/query/SlidingWindows.yml

```diff
@@ -377,6 +377,674 @@ Actors:
+    Parameters:
+      name: StdDevSampPositionBased
+      active: [55]
+      operationMetricsName: StdDevSampPositionBased
+      nopInPhasesUpTo: *MaxPhases
+      Pipeline:
+          [{$setWindowFields: {
+            partitionBy: "$partitionKey",
+            sortBy: {time: 1},
+            output: {stdDevSamp: {
+              $stdDevSamp: "$temp",
+              window: {documents: [-5, 5]}}}}}]
+
+- StdDevSampTimeBased:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: StdDevSampTimeBased
+      active: [56]
+      operationMetricsName: StdDevSampTimeBased
+      nopInPhasesUpTo: *MaxPhases
+      Pipeline:
+          [{$setWindowFields: {
+            partitionBy: "$partitionKey",
+            sortBy: {time: 1},
+            output: {stdDevSamp: {
+              $stdDevSamp: "$temp",
+              window: {
+                range: [-10, 10],
+                unit: "second"
+              }}}}}]
+
+- CovariancePopPositionBased:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: CovariancePopPositionBased
+      active: [57]
+      operationMetricsName: CovariancePopPositionBased
+      nopInPhasesUpTo: *MaxPhases
+      Pipeline:
+          [{$setWindowFields: {
+            partitionBy: "$partitionKey",
+            sortBy: {time: 1},
+            output: {covariancePop: {
+              $covariancePop: "$temp",
+              window: {documents: [-5, 5]}}}}}]
+
+- CovariancePopTimeBased:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: CovariancePopTimeBased
+      active: [58]
+      operationMetricsName: CovariancePopTimeBased
+      nopInPhasesUpTo: *MaxPhases
+      Pipeline:
+          [{$setWindowFields: {
+            partitionBy: "$partitionKey",
+            sortBy: {time: 1},
+            output: {covariancePop: {
+              $covariancePop: "$temp",
+              window: {
+                range: [-10, 10],
+                unit: "second"
+              }}}}}]
+
+- CovarianceSampPositionBased:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: CovarianceSampPositionBased
+      active: [59]
+      operationMetricsName: CovarianceSampPositionBased
+      nopInPhasesUpTo: *MaxPhases
+      Pipeline:
+          [{$setWindowFields: {
+            partitionBy: "$partitionKey",
+            sortBy: {time: 1},
+            output: {covarianceSamp: {
+              $covarianceSamp: "$temp",
+              window: {documents: [-5, 5]}}}}}]
+
+- CovarianceSampTimeBased:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: CovarianceSampTimeBased
+      active: [60]
+      operationMetricsName: CovarianceSampTimeBased
+      nopInPhasesUpTo: *MaxPhases
+      Pipeline:
+          [{$setWindowFields: {
+            partitionBy: "$partitionKey",
+            sortBy: {time: 1},
+            output: {covarianceSamp: {
+              $covarianceSamp: "$temp",
+              window: {
+                range: [-10, 10],
+                unit: "second"
```

Comment:
It seems like using time units is a good idea so that we don't duplicate each test case for non time unit windows again. Just note that the time unit is left out of implementation from the first pushdown PR. I should have disabled pushing down those pipelines.

I created a PR to fix this, we should merge that PR before merging these workloads to measure the correct number. https://github.com/10gen/mongo/pull/14804